boot/zephyr/Kconfig

```diff
@@ -11,6 +11,19 @@ comment "MCUboot-specific configuration options"
 
 source "$(ZEPHYR_NRF_MODULE_DIR)/modules/mcuboot/boot/zephyr/Kconfig"
 
+# Within sdk-nrf NRF security is default
+config NRF_SECURITY
+	bool
+	default y
+	select NRF_OBERON
+	select BOOT_USE_PSA_CRYPTO
+
+# NRF_SECURITY enforces PSA crypt
+config MBEDTLS
+	bool
+	default n if NRF_SECURITY
```

Comment:
`NRF_SECURITY` already exists, it should not be getting defined here again, likewise with MBEDTLS. Default should go in a Kconfig.defconfig file